index.html

```diff
@@ -533,6 +598,80 @@
+                            <span id="piece-size-display">88</span>
+                          </form><br/>
+                          <form>
+                            Board Style<br/>
+                            <div class="radio-list-item">
+                              <input id="board-style-simple" type="radio" name="board-style" value="simple" onClick="radioBoardStyle(this.value)" checked/>
+                              <img src="images/board/board_simple.jpg" class="radio-list-img" height="100" width="200"/>
+                            </div>
+                            <div class="radio-list-item">
+                              <input id="board-style-ornate" type="radio" name="board-style" value="ornate" onClick="radioBoardStyle(this.value)"/>
+                              <img src="images/board/board_ornate.jpg" class="radio-list-img" height="100" width="200"/>
+                            </div>
+                          </form>
+                        </div>
+                        <div class="split-container-right">
+                          <p>
+                            <b>
+                              Interface
+                            </b>
+                            <br/>
+                            <i>Some changes to the interface will require the page to be reloaded.</i><br/>
+                          </p>
+                          <form>
+                            <input id="antialiasing-checkbox" type="checkbox" onClick="checkboxAntialiasing()"/>
+                            Antialiasing (not reccomended for slower devices)
+                            <br/>
+                            <br/>
+                            <input id="hide-send-checkbox" type="checkbox" onClick="checkboxHideSend()"/>
+                            Hide 'Send' button
+                            <br/>
+                            <br/>
+                            <input id="auto-open-chat-checkbox" type="checkbox" onClick="checkboxAutoOpenChat()" checked/>
+                            Show/Hide chat on window resize
+                            <br/>
+                            <br/>
+                            <input id="auto-rotate-checkbox" type="checkbox" onClick="checkboxAutoRotate()" checked/>
+                            Rotate board when player 2
+                          </form>
+                        </div>
+                        
+                        <br/>
+                        <p><br/><i>
+                          Settings made in this dialog are stored local.
+                          If you log in from another device, or if you delete your browser cache,
+                          you will have to set this again.
+                        </i><p/>
+                    </div>
+                </div>
+            </div>
+        </div>
+
```

Comment:
Using default modal now (medium). It will still vanish, if the display is rescaled very small.